matRad_calcLateralParticleCutOff.m

```diff
@@ -173,15 +178,18 @@
     midPos = round(length(vLatX)/2);
     [X,Y] = meshgrid(vLatX,vLatX);
     vRadDist = sqrt(X.^2 + Y.^2);
+    focusIdx = 1;
```

Comment:
Why is the focus index hard coded here? Shouldn't it be a function of energy again?